app/lib/url.rb

```diff
@@ -67,6 +67,16 @@ def self.local_image(image_name, host: nil)
     ActionController::Base.helpers.image_url(image_name, host: host)
   end
 
+  # Creates a deep link URL (for mobile) to a page in the current Forem and it
+  # relies on a UDL server to bounce back mobile users to the local `/r/mobile`
+  # fallback page. More details here: https://github.com/forem/udl-server
+  #
+  # @param path [String] the target path to deep link
+  def self.deep_link(path)
+    target_path = CGI.escape(url("/r/mobile?deep_link=#{path}"))
+    "https://forem-udl-server.herokuapp.com/?r=#{target_path}"
```

Comment:
Non-blocking comment: We could extract `https://forem-udl-server.herokuapp.com` into a constant in this file in case we end up wanting to re-use it or reference it elsewhere (but if we don't think we'll need to do that, then we can probably punt this)

Reply:
We're already playing around with this address, so I definitely agree we should extract this value!